vortex-serde/src/layouts/mod.rs

```diff
@@ -5,7 +5,14 @@ mod pruning;
 #[cfg(test)]
 mod tests;
 
+pub const VERSION: u16 = 1;
```

Comment:
It's a lot easier to check if things are corrupted if we don't default to 0. 0 indicates corrupted file